core/io/compression.cpp

```diff
@@ -181,8 +181,8 @@ int Compression::decompress(uint8_t *p_dst, int p_dst_max_size, const uint8_t *p
 }
 
 /**
-	This will handle both Gzip and Deflat streams. It will automatically allocate the output buffer into the provided p_dst_vect Vector.
-	This is required for compressed data who's final uncompressed size is unknown, as is the case for HTTP response bodies.
+	This will handle both Gzip and Deflate streams. It will automatically allocate the output buffer into the provided p_dst_vect Vector.
+	This is required for compressed data whoes final uncompressed size is unknown, as is the case for HTTP response bodies.
```

Comment:
https://www.yourdictionary.com/whoes :)

This should be "whose".

Reply:
I am making typos in  corrections ...wow  🤦‍♂️

Reply:
Apparently who's is also correct